cmd/manager/main.go

```diff
@@ -57,58 +57,81 @@ func main() {
 
 	// Create a new Cmd to provide shared dependencies and start components
 	log.Info("setting up manager")
-	mgr, err := manager.New(cfg, manager.Options{MetricsBindAddress: "0"})
+	// watchMgr is namespace scoped. it is used by the controllers to watch the resources owned by
+	// the controllers present in the migration namespace
+	watchMgr, err := manager.New(cfg, manager.Options{MetricsBindAddress: "0", Namespace: "openshift-migration"})
 	if err != nil {
 		log.Error(err, "unable to set up overall controller manager")
 		os.Exit(1)
 	}
 
+	// unscopedMgr is not namespace scoped. it is used by controllers to list resources
+	// outside of the migration namespace
+	unscopedMgr, err := manager.New(cfg, manager.Options{MetricsBindAddress: "0"})
+	if err != nil {
+		log.Error(err, "unable to set up unscoped manager")
+		os.Exit(1)
+	}
+
 	log.Info("Registering Components.")
 
 	// Setup Scheme for all resources
 	log.Info("setting up scheme")
-	if err := apis.AddToScheme(mgr.GetScheme()); err != nil {
+	if err := apis.AddToScheme(watchMgr.GetScheme()); err != nil {
 		log.Error(err, "unable to add K8s APIs to scheme")
 		os.Exit(1)
 	}
-	if err := velerov1.AddToScheme(mgr.GetScheme()); err != nil {
+	if err := velerov1.AddToScheme(watchMgr.GetScheme()); err != nil {
 		log.Error(err, "unable to add Velero APIs to scheme")
 		os.Exit(1)
 	}
-	if err := imagescheme.AddToScheme(mgr.GetScheme()); err != nil {
+	if err := imagescheme.AddToScheme(watchMgr.GetScheme()); err != nil {
 		log.Error(err, "unable to add OpenShift image APIs to scheme")
 		os.Exit(1)
 	}
-	if err := appsv1.AddToScheme(mgr.GetScheme()); err != nil {
+	if err := appsv1.AddToScheme(watchMgr.GetScheme()); err != nil {
 		log.Error(err, "unable to add OpenShift apps APIs to scheme")
 		os.Exit(1)
 	}
-	if err := routev1.AddToScheme(mgr.GetScheme()); err != nil {
+	if err := routev1.AddToScheme(watchMgr.GetScheme()); err != nil {
```

Comment:
will all of these add to schemes will also be required for unscopedMgr? Wondering what will happen if the dstClient tried to get routes?